client/src/components/Markdown/Sections/MarkdownGalaxy.vue

```diff
@@ -158,10 +162,10 @@ watch(
                 :path="args.path" />
             <HistoryDatasetAsTable
                 v-else-if="name == 'history_dataset_as_table'"
-                :compact="argToBoolean(args, 'compact', false)"
+                :compact="compact"
                 :dataset-id="args.history_dataset_id"
                 :footer="args.footer"
-                :show-column-headers="argToBoolean(args, 'show_column_headers', true)"
+                :show-column-headers="showColumnHeaders"
                 :title="args.title"
                 :path="path" />
```

Comment:
```suggestion
                :path="args.path" />
```

This should also get rid of this console error:

`Property or method "path" is not defined on the instance but referenced during render. Make sure that this property is reactive, either in the data option, or for class-based components, by initializing the property. See: https://v2.vuejs.org/v2/guide/reactivity.html#Declaring-Reactive-Properties.
`

Reply:
Thank you!